services/QuillLMS/client/app/bundles/Grammar/components/grammarActivities/intro.tsx

```diff
@@ -85,7 +86,7 @@ export default class Intro extends React.Component<IntroProps, IntroState> {
     const { activity, language, availableLanguages } = this.props
     const { showLandingPage, } = this.state
     const translatedText = this.translatedText()
-    const showTranslatedButton = language && availableLanguages?.includes(language)
+    const showTranslatedButton = language && language !== ENGLISH && availableLanguages?.includes(language)
```

Comment:
This is almost certainly beyond the scope of this PR, so I think we should probably merge this PR as-is, but I wonder if we should consider a broader refactor that doesn't special-case `ENGLISH` for translation triggers.  (I recognize that we actually do have different behavior for buttons for non-English languages, but I wonder if this is the right place to place that logic.)

Anyway, I think that's a broader discussion, one that I'll add to the next engineering meeting agenda for discussion.